src/components/Calculator.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Calculator.css';
```

Comment:
- [ ] You do not have a Calculator.css and this is throwing an error, kindly fix it 
![image](https://user-images.githubusercontent.com/62705947/221053705-314a487e-a255-4e7f-bb94-1f2034e029cd.png)